src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -314,7 +314,8 @@ void PolicyManagerImpl::StartPTExchange() {
       if (RequestPTUpdate() && !timer_retry_sequence_.is_running()) {
         // Start retry sequency
         const int timeout_sec = NextRetryTimeout();
-        LOG4CXX_DEBUG(logger_, "Start retry sequence timeout = " << timeout_sec);
+        LOG4CXX_DEBUG(logger_,
+                      "Start retry sequence timeout = " << timeout_sec);
```

Comment:
This change is irrelevant (fixes a formatting issue) and already exists in develop.